src/main/webapp/js/utils/table-utils.jsx

```diff
@@ -56,16 +59,51 @@ const getTableCell = (item, props) => {
   );
 };
 
+const getTableFilter = (filter, onChange, format, fieldType) => {
+  let Filter;
+  switch (fieldType) {
+    case ENUM:
+      Filter = EnumFilter;
+      break;
+    default:
+      Filter = TextFilter;
+  }
+
+  return (
+    <Filter filter={filter} onChange={onChange} format={format} />
+  );
+};
+
 const getTableColumn = (props) => {
   const {
-    name, displayName, filterable,
+    name, displayName, filterable, format, fieldType,
   } = props;
 
   return {
     Header: displayName,
     accessor: name,
     filterable,
     Cell: item => getTableCell(item, props),
+    filterMethod: (filter, row) => {
+      let filterValue = null;
+      if (filter.value === 'ALL') {
+        filterValue = true;
+      }
+      if (fieldType === 'ENUM' && format) {
+        _.forEach(format.split(','), (val) => {
+          const chunks = val.split(':');
+          if (filter.value === chunks[0]) {
+            filterValue = (row[filter.id] === chunks[0]);
+          }
+        });
+      } else if (fieldType !== 'ENUM') {
+        filterValue = row[filter.id].startsWith(filter.value);
```

Comment:
I think it will be best to use the method from Mots:
`_.toString(row[filter.id]).toLowerCase().includes(filter.value.trim().toLowerCase())`

Reply:
I've seen this method. Before any changes in NEWEBODAC the text filter was working as 'startsWith', but this method returns true if text includes the filter value. I've changed it, but just want to make sure it works as expected.